FVCalc.java

```diff
@@ -2,5 +2,11 @@
 public class FVCalc {
 	public static void main(String[] args){
 		// Replace this comment with your code
+		double a = Double.parseDouble(args[0]);
+		double b = Double.parseDouble(args[1]);
+		double c = Double.parseDouble(args[2]);
```

Comment:
It is better to work with int rather then double, whenever it is possible, due to double inaccuracy.